api/src/test/java/org/openmrs/module/datafilter/impl/api/db/hibernate/AccessInterceptorTest.java

```diff
@@ -208,7 +209,7 @@ public void onLoad_shouldFailIfTheAuthUserIsNotAllowedToViewThePrivilegedObsGett
 		ee.expectMessage(equalTo(ImplConstants.ILLEGAL_RECORD_ACCESS_MESSAGE));
 		when(AccessUtil.getEncounterTypeId(Matchers.eq(encounterId))).thenReturn(encounterTypeId);
 		interceptor.onLoad(new Obs(), obsId, new Object[] { new Encounter(encounterId) }, new String[] { "encounter" },
-		    new Type[] { new ManyToOneType(null, null) });
+		    new Type[] { new ManyToOneType((TypeFactory.TypeScope) null, null) });
```

Comment:
We now have one more constructor on hibernate5 w/ 2 args so, casted the 1st arg to resolve to the right one